scripts/qa_crowbarsetup.sh

```diff
@@ -525,7 +525,7 @@ function addsp3testupdates()
     add_mount "SLES11-SP3-Updates-test" \
         $distsuseip':/dist/ibs/SUSE:/Maintenance:/Test:/SLE-SERVER:/11-SP3:/x86_64/update/' \
         "$tftpboot_repos_dir/SLES11-SP3-Updates-test/" "sp3tup"
-    [ -n "$hacloud" ] && add_mount "SLE11-HAE-SP3-Updates-test" \
```

Comment:
What would you think about changing that to `[0, 1, ...]`? In other words, set a default value of `0` for `hacloud`, replacing the empty string. Within this repository, the necessary changes wouldn't be all that bad. Is `hacloud` used outside this repository?